pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/upsertcompaction/UpsertCompactionTaskExecutor.java

```diff
@@ -108,7 +110,11 @@ protected SegmentConversionResult convert(PinotTaskConfig pinotTaskConfig, File
             .setSegmentName(segmentName).build();
 
     long endMillis = System.currentTimeMillis();
-    LOGGER.info("Finished task: {} with configs: {}. Total time: {}ms", taskType, configs, (endMillis - startMillis));
+    long validDocIdsCount = validDocIds.stream().count();
+    LOGGER.info("Finished task: {} with configs: {}. Total time: {}ms. "
+            + "Total docs before compaction: {}. Total docs after compaction: {}. Valid doc IDs count: {}",
+            taskType, configs, (endMillis - startMillis), segmentMetadata.getTotalDocs(), totalDocsAfterCompaction,
+            validDocIdsCount);
```

Comment:
Thanks for this! I was planning to add a metric to track number of rows compacted. This was to show business impact of compaction process. If it's not much, can you do that here itself? Thanks!!